api/__init__.py

```diff
@@ -0,0 +1,40 @@
+import requests
+
+DEFAULT_CONTENT_TYPE = "application/json"
+DEFAULT_ACCEPT = "application/vnd.ceph.api.v1.0+json"
+DEFAULT_VERIFY = True
+
+
+class Api:
+    def __init__(self, url, api):
+        requests.packages.urllib3.disable_warnings(
+            requests.packages.urllib3.exceptions.InsecureRequestWarning
+        )
+
+        self.url = f"{url}/${api}"
+
+    def get(self):
+        pass
+
+    def post(
+        self,
+        data,
+        verify=DEFAULT_VERIFY,
+        content_type=DEFAULT_CONTENT_TYPE,
+        accept=DEFAULT_ACCEPT,
+    ):
+        headers = {
```

Comment:
Accept more headers for future use.

Reply:
+1. Instead of these values, will accept header